app/javascript/components/public_health/PatientsTable.js

```diff
@@ -518,6 +520,33 @@ class PatientsTable extends React.Component {
     return <EligibilityTooltip id={rowData.id.toString()} report_eligibility={reportEligibility} inline={false} />;
   };
 
+  renderFollowUpTooltip = data => {
+    const flaggedForFollowUp = data.value;
+    const rowData = data.rowData;
+    return (
+      <React.Fragment>
+        {flaggedForFollowUp.follow_up_reason && (
+          <React.Fragment>
+            <span key={`flagged-icon-${rowData.id}`} data-for={`flagged-${rowData.id}`} data-tip="">
+              <div className="text-center ml-0">
+                <i className="fa-fw fas fa-flag"></i>
+              </div>
+            </span>
+            <ReactTooltip key={`flagged-tooltip-${rowData.id}`} id={`flagged-${rowData.id}`} multiline={true} place="right" type="dark" effect="solid">
+              <div>
+                Monitoree is flagged for follow-up.
+                <br />
+                {flaggedForFollowUp.follow_up_reason}
+                {flaggedForFollowUp.follow_up_note && flaggedForFollowUp.follow_up_note.length < 75 && <span>{': ' + flaggedForFollowUp.follow_up_note}</span>}
+                {flaggedForFollowUp.follow_up_note.length >= 75 && <span>{': ' + flaggedForFollowUp.follow_up_note.slice(0, 75) + ' ...'}</span>}
```

Comment:
If `flaggedForFollowUp.follow_up_note` ends up being nil then this will break. I think that here we either need to add the `flaggedForFollowUp.follow_up_note &&` check like the line above, or if this really is never expected to be null then we need to validate for it in the patients model.